frontend/src/modules/organizers/components/OrganizerAddForm.tsx

```diff
@@ -0,0 +1,101 @@
+import { Button, Flex, Spacer } from '@chakra-ui/react';
+import { Field, FieldProps, Form, Formik } from 'formik';
+import { OrganizerDTO, OrganizerDTOTypeEnum } from 'generated-api';
+import { useEffect } from 'react';
+import * as Yup from 'yup';
+import { FormLayout, Input, Select } from '../../../shared/components/form';
+import { useToast } from '../../../shared/hooks';
+import { useOrganizers } from '../hooks';
+
+interface OrganizerAddFormProps {
+  onComplete: (organizer: OrganizerDTO) => void;
+}
+
+export const OrganizerAddForm = ({ onComplete }: OrganizerAddFormProps) => {
+  const { addOrganizer } = useOrganizers();
+  const toast = useToast();
+
+  const onSubmit = (organizer: OrganizerDTO) => {
+    addOrganizer.mutate(organizer);
```

Comment:
it would also be nice to refetch the organizer list once an organizer is added successfully

same thing when registering an account in the accounts page, we should refetch all accounts once the registration is successful

Reply:
Yes yes, there's a refetch on Accounts and Organizers on their useEffect blocks, all of them re-renders on each Edit, Add, and Delete updates 😁